datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -1007,6 +1007,10 @@ private void validateShape(SdxClusterShape shape, String runtime, DetailedEnviro
                         + MEDIUM_DUTY_REQUIRED_VERSION + " and not " + runtime);
             }
         } else if (SdxClusterShape.ENTERPRISE.equals(shape)) {
+            if (entitlementService.enterpriseSdxDisabled(Crn.safeFromString(environment.getCreator()).getAccountId())) {
+                validationBuilder.error(String.format("Provisioning an enterprise data lake cluster is disabled for %s. " +
+                        "Contact Cloudera support to enable this scale for the account.", environment.getCloudPlatform()));
```

Comment:
adding cloudplatform to the message is not necessary, if entitlement is enabled, then cloudplatform does not matter